### PR TITLE
DT-426 modified -p option to work with multiple usage, no warning issued

### PR DESCRIPTION
**Pull request by SergioJanrain**

### CODE REVIEW

File: janrain/capture/cli.py [hunk 1]
```diff
@@ -114,6 +114,13 @@ def init_api(self, api_class=None):
         else:
             return Api(credentials['apid_uri'], defaults)
 
+# parameterGen flattens the parameters list if multiple -p is used
+def parameterGen(params):
```
Comment (Janrain-Colin) [inline janrain/capture/cli.py]: Please use `snake_case` function and variable names.

**Comment by Janrain-Colin** [comment]
Multiple -p options still overwrite previous ones (or I'm confused, which is possible).

File: janrain/capture/cli.py [hunk 2]
```diff
@@ -114,6 +114,13 @@ def init_api(self, api_class=None):
...
+    for s in params:
+        if isinstance(s, list):
+            for item in s:
+                yield item.split("=", 1)
+        else: yield s.split("=", 1)
```
Comment (Janrain-Colin) [inline janrain/capture/cli.py]: I also have some thoughts about how this function could be generalized.

**Comment by SergioJanrain** [comment]
By adding the append action to the -p arg, it now stores its arguments as a list of lists; each group of args associated with a usage of -p is grouped into its own sublist. Let's say a user runs with:
`capture-api -k customer -p type_name=user uuid=whatever -p type_name=banana`
This produces `args.parameters = [['type_name=user', 'uuid=whatever'], ['type_name=banana']]`. After flattening, this yields `[[type_name, user], [uuid, whatever], [type_name=banana]]`. When the tuples are added to the `kwargs` dictionary as key/value pairs, the second usage of type_name will overwrite the previous usage, but uuid will persist, so `kwargs = { 'type_name': 'banana', 'uuid': 'whatever'}`.

I can have it issue a warning if an existing key is encountered during the creation of `kwargs`, if you'd like.

**Comment by ebeesonjanrain** [comment]
:shipit:

**Pull request merged**